Chummer/data/gear.xml

```diff
@@ -1531,6 +1531,51 @@
         <damage>-1</damage>
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damage>6P Acid</damage>
```

Comment:
This will not work. The `<damage>` node modifies the existing damage number of a weapon, and the `<damagetype>` node modifies the existing damage type of a weapon (stun or physical). You are probably looking for `<damagereplace>`, which outright replaces the damage code of a weapon.